src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedTryDepthCheck.java

```diff
@@ -48,7 +48,27 @@
  *   &lt;property name=&quot;max&quot; value=&quot;3&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>Example of violation:</p>
+ * <pre>
+ * try { // ...
+ *     try { // ...
+ *         try { // violation
```

Comment:
in config max value is set to 3, it should not be a violation, isn't it? In PR description you have max value = 2